modules/core/src/debug/loggers.ts

```diff
@@ -103,7 +109,7 @@ export const getLoggers = log => ({
     logState.attributeUpdateMessages.push(message);
   },
   'attribute.updateEnd': (attribute, numInstances) => {
-    const timeMs = Math.round(Date.now() - logState.attributeUpdateStart);
+    const timeMs = Math.round(Date.now() - logState.attributeUpdateStart!);
```

Comment:
I much prefer ts-expect-error compared to !. The latter is so hard to spot and it has bitten me so hard in the past.